lms/envs/devstack.py

```diff
@@ -175,17 +175,29 @@ def should_show_debug_toolbar(request):
 ########################## Course Discovery #######################
 LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
 COURSE_DISCOVERY_MEANINGS = {
```

Comment:
BTW why do we overwriting `COURSE_DISCOVERY_MEANINGS` If we use the same parameters as common.py set?

Reply:
Removed from `devstack.py`